tasks/25_rot13-encoder/rot13-encoder.js

```diff
@@ -0,0 +1,6 @@
+export function rot13Encoder(str) {
+  return str.replace(/[A-Z]/g, (c) => String.
+      fromCharCode(('A'.charCodeAt()
+      +
+      (c.charCodeAt() - 'A'.charCodeAt() + 13)%26)));
```

Comment:
Please do not use extra lines if there is no need for that

```
return str.replace(/[A-Z]/g, (c) => String.fromCharCode(('A'.charCodeAt(0) +
    (c.charCodeAt(0) - 'A'.charCodeAt(0) + 13)%26)));
```

Reply:
Fixed the lines and added spaces so it would be more readable.